### PR TITLE
Id pool fix

### PR DESCRIPTION
**Pull request by brnkhy**
fix a bug where VectorLayerVisualizer didn't add first feature's id to idpool

@greglemonmapbox @atripathi-mb

### CODE REVIEW

File: sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs [hunk 1]
```diff
@@ -450,6 +450,10 @@ private VectorFeatureUnity GetFeatureinTileAtIndex(int index, UnityTile tile, Ve
 		/// <param name="featureId">Feature identifier.</param>
 		private bool ShouldSkipProcessingFeatureWithId(ulong featureId, UnityTile tile, VectorLayerVisualizerProperties layerProperties)
 		{
+			if (_activeIds.Contains(featureId))
```
Comment (atripathi-mb) [inline sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs]: Remove Debug statements

File: sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs [hunk 2]
```diff
@@ -727,6 +731,7 @@ public override void OnUnregisterTile(UnityTile tile)
 			//removing ids from activeIds list so they'll be recreated next time tile loads (necessary when you're unloading/loading tiles)
 			if (_idPool.ContainsKey(tile))
 			{
+				Debug.Log(_idPool[tile].Count + " " + _activeIds.Count);
```
Comment (atripathi-mb) [inline sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs]: Remove debug statements

File: sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs (hunk 1 above)
Comment (brnkhy) [reply sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs]: removed

File: sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs (hunk 2 above)
Comment (brnkhy) [reply sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs]: removed

**Pull request merged**